frontend/src/components/Team/TeamCard.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="row">
+    <div class="col-sm" v-for="teamMate in teamMate" :key="teamMate.id">
+      <v-col class="pl-2 pr-2 pt-2">
+        <v-card tile>
+          <v-col align="center" class="mt-5">
+            <v-avatar size="100">
+              <v-img v-bind:src="teamMate.picture"></v-img>
```

Comment:
```suggestion
              <v-img :src="teamMate.picture"></v-img>
```
`:foo` is the same as `v-bind:foo`